backend/typescript/package.json

```diff
@@ -68,8 +68,10 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-prettier": "^3.3.1",
     "jest": "^27.0.4",
+    "moment": "^2.20.1",
```

Comment:
I believe `moment` is deprecated, we [could use](https://momentjs.com/docs/#/-project-status/recommendations/) `Day.js` or `date-fns` ?

Reply:
updated to use dayjs